.github/workflows/live/afp-ids.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#set -e
+set -x
+
+TPACKET=$1
+RUNMODE=$2
+
+uname -a
+#ip r
```

Comment:
Any reason not to leave this? I feel like its one of those things we might need when this breaks, then we'll have to create a dummy branch, push, etc :)

Reply:
added back in